index.html

```diff
@@ -336,11 +336,51 @@ <h2 >About me</h2>
             </div>
             
         </section>
-        <section class="form">
+        <footer class="form">
             <div class="txtF">
                 <p class="primary_txt_F"> I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
             </div>
-        </section>
-
+            <div class="form_f">
+                <form action="https://formspree.io/f/mgeqgykl" method="post">
+                    <input type="text"  id="fullname"  required class="fname" name="fullname" placeholder="Full name" maxlength="30">
+                    
+                   <input type="email" id="fullname" required class="email" name="email" placeholder="Email address">
+                    
+                    <textarea class="txt_area" id="fullname" required placeholder="Enter text here" maxlength="500"></textarea>
+                        <button type="button"  class="button"><p class="btx1">Get in touch</p></button>
```

Comment:
Good job correctly implementing the hover state for the button 👍. According to the Figma template, an active (pressed) state should be styled also. I think it will make the button more visually interactive if this is added too.


<img width="484" alt="pressed" src="https://user-images.githubusercontent.com/84930355/206054534-e55aadd0-d403-40b2-a117-056f02a9a70a.png">